### PR TITLE
COMMERCE-9268 Integration test implementation for the PaymentMethodGroupRelOrderTypeResourceTest class

### PR DESCRIPTION
**Pull request by ghost**
https://issues.liferay.com/browse/COMMERCE-9268

### CODE REVIEW

**Comment by CrescenzoRegaLF** [comment]
ci:test:sf

**Comment by CrescenzoRegaLF** [comment]
ci:test:relevant

File: modules/apps/commerce/headless/headless-commerce/headless-commerce-admin-channel-test/src/testIntegration/java/com/liferay/headless/commerce/admin/channel/resource/v1_0/test/PaymentMethodGroupRelOrderTypeResourceTest.java [hunk 1]
```diff
@@ -15,15 +15,300 @@
...
+	private CommerceCurrency _commerceCurrency;
+
+	@DeleteAfterTestRun
+	private List<CommerceOrderType> _commerceOrderTypes = new ArrayList<>();
+
+	@Inject
+	private CommercePaymentMethodGroupRelQualifierLocalService
+		_commercePaymentMethodGroupRelQualifierLocalService;
+
+	@DeleteAfterTestRun
```
Comment (luca-pellizzon) [inline modules/apps/commerce/headless/headless-commerce/headless-commerce-admin-channel-test/src/testIntegration/java/com/liferay/headless/commerce/admin/channel/resource/v1_0/test/PaymentMethodGroupRelOrderTypeResourceTest.java:301]: Are you sure this also works with collections?

Comment (CrescenzoRegaLF) [reply modules/apps/commerce/headless/headless-commerce/headless-commerce-admin-channel-test/src/testIntegration/java/com/liferay/headless/commerce/admin/channel/resource/v1_0/test/PaymentMethodGroupRelOrderTypeResourceTest.java:301]: yes I've seen it used in other test classes too

**Comment by luca-pellizzon** [comment]
ci:forward

**Pull request closed without merging**